backend/wallet/services/transaction.py

```diff
@@ -59,50 +58,59 @@ class P2MTxnRegistrationError(Exception):
 #               currency: XUS,
 #               recipient_signature: None,
 #               receiver_address: tdm1paue4j3fqr6nzl5xglqr337surful4nmrx6kn9fgmh5qz6
```

Comment:
do we really need this comment?

Reply:
it's not a must, but since we spent more than an hour trying to understand how the data should look like, I believe it is good documenting this for future reference